metric_tank/metric_tank.go

```diff
@@ -28,13 +28,15 @@ var (
 	showVersion = flag.Bool("version", false, "print version string")
 	dryRun      = flag.Bool("dry", false, "dry run (disable actually storing into cassandra")
```

Comment:
now that we have better terminology, i think it would be better to use a var named primaryMode bool default true, or something.